tests/test_mux.py

```diff
@@ -584,14 +584,14 @@ def multi_nexthop_test(self, dvs, dvs_route, asicdb, appdb, route, neighbors, ma
                         print("setting %s to %s" % (mux_ports[toggle_index], ACTIVE))
                         self.set_mux_state(appdb, mux_ports[toggle_index], ACTIVE)
                         if start[keep_index] == ACTIVE:
-                            self.check_route_nexthop(dvs_route, asicdb, route, neighbors[keep_index])
+                            self.check_route_nexthop(dvs_route, asicdb, route, neighbors[0])
```

Comment:
Why is existing test changed?

Reply:
The logic changed, so this test was failing. Each time we update the route, we'll be programming the route to point to the first active neighbor. Before this, we were keeping whatever nexthop was previously programmed.